state/txn.go

```diff
@@ -67,6 +70,86 @@ func newTxn(snapshot readSnapshot) *Txn {
 	}
 }
 
+// GetDumpTree function returns accounts based on the selected criteria.
+func (txn *Txn) GetDumpTree(dumpObject *Dump, opts *DumpInfo, deleteEmptyObjects bool) ([]byte, error) {
+	if err := txn.CleanDeleteObjects(deleteEmptyObjects); err != nil {
+		return nil, err
+	}
+
+	var (
+		nextKey         []byte
+		startValid      = opts.Start == nil || len(opts.Start) == 0 || bytes.Equal(opts.Start, types.EmptyRootHash.Bytes())
+		committedIradix = txn.GetRadix().Commit()
```

Comment:
```suggestion
		committedIradix = txn.txn.Commit()
```
I think it can be do it this way, txn.txn is *iradix.Txn. There is an example in func (txn *Txn) Commit(deleteEmptyObjects bool) ([]*Object, error) {

Reply:
It should be the same.

func (txn *Txn) GetRadix() *iradix.Txn {
	return txn.txn
}

Reply:
It is the same, but it's better to avoid function call if possible.